src/device/mod.rs

```diff
@@ -2728,10 +2728,13 @@ mod tests {
 
         let (sender, _receiver) = tokio::sync::broadcast::channel(1);
 
-        let mut providers = HashSet::<telio_model::api_config::EndpointProvider>::new();
-        providers.insert(telio_model::api_config::EndpointProvider::Stun);
-        providers.insert(telio_model::api_config::EndpointProvider::Upnp);
-        providers.insert(telio_model::api_config::EndpointProvider::Local);
+        let providers = vec![
```

Comment:
nit: you don't need a vector, an array should also work but you can use `maplit::hashset` instead of creating temporary collection.

Reply:
Fixed